packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -75,6 +75,11 @@ export default class Deploy extends Command {
       env: 'SHOPIFY_HYDROGEN_FLAG_FORCE',
       required: false,
     }),
+    'skip-verification': Flags.boolean({
```

Comment:
I wonder if we should follow the pattern used by some of the other flags and call this `verification-check` and use `allowNo: true`. Similar to `lockfileCheck`: https://github.com/Shopify/hydrogen/blob/main/packages/cli/src/lib/flags.ts#L107-L113

It'd be like:

```ts
'verification-check': Flags.boolean({
  allowNo: true,
  default: true,
  description: 'Ensures the Oxygen deployment is routable. Defaults to true, use `--no-verification-check` to disable.',
 env: 'SHOPIFY_HYDROGEN_FLAG_VERIFICATION_CHECK',
})
```

What do you think?